vscode/polar-lang/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "oso",
+  "publisher": "osohq",
+  "displayName": "oso",
+  "description": "Syntax highlighting for oso .polar files",
+  "icon": "oso_logo.png",
+  "version": "0.5.0",
+  "engines": {
+    "vscode": "^1.43.0"
```

Comment:
is this a hard requirement? it appears that 1.43 only came out in February 2020

Reply:
vscode auto updates so I don't think this matters. This is what it put in when the extension was generated so maybe it does depend on something in this release.

Reply:
Okay, WFM